apps/docs/app/routes/_index/HeroSection.tsx

```diff
@@ -14,53 +13,60 @@ import {
-            alignItems="center"
-            gap={4}
+            flexDirection="column"
+            color="text.inverted"
+            flex={["auto", "55%"]}
           >
-            <Flex
-              flexDirection="column"
-              color="text.inverted"
-              flex={["auto", "55%"]}
+            <Heading
+              as="h1"
+              variant="2xl"
+              color="text.tertiary"
+              className="dark"
             >
-              <Heading as="h1" variant="2xl">
-                Welcome to Spor
-              </Heading>
-              <Text variant="sm" color="text.tertiary" className="dark">
-                Spor is Vy's design system for digital products.
-              </Text>
-              <Box marginBottom={[10, null, 0]}>
-                <Button
-                  variant="primary"
-                  display={["flex", "inline-flex"]}
-                  className="dark"
-                  as={Link}
-                  size="lg"
-                  onClick={() => navigate("/guides/introduction")}
-                  rightIcon={<ArrowRightFill30Icon />}
-                  marginTop={4}
-                  width={["100%", null, "fit-content"]}
-                >
-                  Get started
-                </Button>
-              </Box>
-            </Flex>
-            <Flex flex={["auto", null, "45%"]}>
-              <Image
-                src={`/illustrations/front-page-illustration-dark.svg`}
-                alt="An illustration of a person looking out over a landscape with trees, houses, buses, bicycles and roads"
-                width="100%"
-                aspectRatio="4 / 3"
-                maxWidth={["400px", null, "none"]}
-              />
-            </Flex>
+              Welcome to Spor
+            </Heading>
+            <Text variant="sm" color="text.tertiary" className="dark">
+              Spor is Vy's design system for digital products.
+            </Text>
+            <Box marginBottom={[10, null, 0]}>
+              <Button
+                className="light"
+                variant="secondary"
+                display={["flex", "inline-flex"]}
+                as={Link}
+                size="lg"
+                onClick={() => navigate("/guides/introduction")}
+                rightIcon={<ArrowRightFill30Icon />}
+                marginTop={4}
+                width={["100%", null, "fit-content"]}
+              >
+                Get started
+              </Button>
+            </Box>
           </Flex>
-        </Container>
-      </Flex>
-    </ColorModeProvider>
+          <Flex flex={["auto", null, "45%"]}>
+            <Image
+              src={`/illustrations/front-page-illustration-dark.svg`}
+              alt="An illustration of a person looking out over a landscape with trees, houses, buses, bicycles and roads"
+              width="100%"
+              aspectRatio="4 / 3"
+              maxWidth={["400px", null, "none"]}
```

Comment:
vi prøver å bruke dynamisk unit for alle størrelse både i spor og main-frontend for å sikre en bedre UU zooming response, så foreslå jeg her en `maxWidth={["25rem", null, "none"]}` for kontinuitet